catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-forms.less

```diff
@@ -0,0 +1,36 @@
+@{customElementNamespace}search-form {
+
+    display: inline-block;
+    padding: @minimumSpacing;
+    margin: @minimumSpacing;
+    width: 300px;
+    height: 125px;
+    text-align: left;
+    vertical-align: top;
+
+    .search-form-title {
+        padding-bottom: @minimumSpacing;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+
+    .search-form-contents {
+        display: block;
+    }
+
+    .search-form-label {
+        font-weight: bold;
+    }
+}
+
+@{customElementNamespace}search-form.is-static {
+    display: inline-block;
+    padding: @minimumSpacing;
+    margin: @minimumSpacing;
+    width: 300px;
+    height: 125px;
+    text-align: center;
```

Comment:
I think we can get rid of the css that isn't text-align center, since that is already covered by the less above for `search-form`s.

Reply:
Done